storage/badger/events_test.go

```diff
@@ -0,0 +1,42 @@
+package badger_test
+
+import (
+	"testing"
+
+	"github.com/dgraph-io/badger/v2"
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/utils/unittest"
+
+	badgerstorage "github.com/onflow/flow-go/storage/badger"
+)
+
+func TestEventStoreRetrieve(t *testing.T) {
+	unittest.RunWithBadgerDB(t, func(db *badger.DB) {
+		store := badgerstorage.NewEvents(db)
+
+		blockID := unittest.IdentifierFixture()
+		txID := unittest.IdentifierFixture()
+		expected := []flow.Event{unittest.EventFixture(flow.EventAccountCreated, 0, 0, txID)}
+
+		// store event
+		err := store.Store(blockID, expected)
+		require.NoError(t, err)
+
+		// retrieve by blockID
+		actual, err := store.ByBlockID(blockID)
```

Comment:
test `ByBlockID`, `ByBlockIDEventType` and `ByBlockIDTransactionID` returns NotFound for non-existing Events

Reply:
These all return `nil` errors with an empty `[]Event` array, should this be changed to return `storage.ErrNotFound`?